src/wasm-lib/kcl/src/ast/types.rs

```diff
@@ -1598,8 +1763,8 @@ impl VariableDeclaration {
         None
     }
 
-    /// Returns a value that includes the given character position.
-    pub fn get_mut_value_for_position(&mut self, pos: usize) -> Option<&mut Expr> {
+    /// Returns an Expr that includes the given character position.
+    pub fn get_mut_expr_for_position(&mut self, pos: usize) -> Option<&mut Expr> {
```

Comment:
I know this is just a draft PR, but it'll be much easier to review if you do renames like this in a separate PR :)

Reply:
#4163